src/server/game/Spells/SpellEffects.cpp

```diff
@@ -4422,7 +4422,7 @@ void Spell::EffectPullTowardsDest(SpellEffIndex effIndex)
     float distXY = unitTarget->GetExactDist(pos);
     float distZ = pos->GetPositionZ() - unitTarget->GetPositionZ();
 
-    float speedXY = m_spellInfo->Effects[effIndex].MiscValue / 10.0f;
+    float speedXY = m_spellInfo->Effects[effIndex].MiscValue ? m_spellInfo->Effects[effIndex].MiscValue / 10.0f : 30.0f;
```

Comment:
is the 30.0f some sort of standard value ?

Reply:
https://github.com/TrinityCore/TrinityCore/blob/6835fb9dcb418f8ced09cc1c8244d646cd2abb98/src/server/game/Spells/SpellEffects.cpp#L4400 yes. It's just missing here because nobody was thinking about that it's possible here as well